server/src/main/scala/actors/FlightMessageConversion.scala

```diff
@@ -81,10 +81,10 @@ object FlightMessageConversion {
     Arrival(
       Operator = flightMessage.operator.getOrElse(""),
       Status = flightMessage.status.getOrElse(""),
-      EstDT = apiFlightDateTime(flightMessage.estimated),
-      ActDT = apiFlightDateTime(flightMessage.touchdown),
-      EstChoxDT = apiFlightDateTime(flightMessage.estimatedChox),
-      ActChoxDT = apiFlightDateTime(flightMessage.actualChox),
+      Estimated = flightMessage.estimated.getOrElse(0),
```

Comment:
I think making these options would make more sense.

Reply:
100% agree

Reply:
If you are up for it, we should probably make most of the fields options, only flight code, origin, terminal and schedule date are compulsory.

Reply:
Ok, Thanks.